gammapy/spectrum/observation.py

```diff
@@ -827,30 +827,41 @@ def stack_counts_spectrum(counts_spectrum_list):
             quality=stacked_quality,
         )
 
+    #modif AL : correct bug on alpha
     def stack_backscal(self):
         """Stack ``backscal`` for on and off vector."""
         nbins = self.obs_list[0].e_reco.nbins
         bkscal_on = np.ones(nbins)
         bkscal_off = np.zeros(nbins)
-
+        
+        alpha_sum=0.0
+        
         for obs in self.obs_list:
             bkscal_on_data = obs.on_vector._backscal_array.copy()
 
             bkscal_off_data = obs.off_vector._backscal_array.copy()
             bkscal_off += (bkscal_on_data / bkscal_off_data) * obs.off_vector.counts_in_safe_range.value
 
+            alpha_sum += obs.alpha * obs.off_vector.counts_in_safe_range.sum()
+            
```

Comment:
In this line you assume that ``alpha`` is not a function of energy, right? This need not be the case, and is not covered by the test you added. I don't fully understand the changes you made, but there could be a problem here